File: Proyectos_c/menu_while_dowhile.c
#include <stdio.h>
int main (){
    int base;
    int altura;
    int radio;
    int lado;
    float pi=3.14159265359;

    while(1){
    printf("Hola! puedo calcular el area de tres figuras distintas, triangulo, circulo y cuadrado.\n");
    printf("Selecciona una figura\n");
    printf("1.Triangulo\n2.Circulo\n3.Cuadrado\n4.Salir\n");
    int opcion;
    scanf("%d" , &opcion);
    if (opcion == 1){
        printf("Ingresa la altura del triangulo\n");
        scanf("%d",&altura);
        printf("Ingresa la base del triangulo\n");
        scanf("%d",&base);
        int AreaDeTriangulo=(base*altura)/2;
        printf("El area de tu triangulo es: %d\n" , AreaDeTriangulo);
    }else if (opcion == 2){
        printf("Ingresa el radio de tu circulo\n");
        scanf("%d", &radio);
        int AreaDeCirculo=(pi*(radio*radio));
        printf("El area de tu circulo es %d\n" , AreaDeCirculo);

    }else if (opcion == 3){
        printf("Ingresa la medida de uno de los lados de tu cuadrado\n");
        scanf("%d", &lado);
        int AreaDeCuadrado=lado*lado;
        printf("El area de tu cuadrado es %d\n", AreaDeCuadrado);
    }else if (opcion == 4){
        break;
    }else{
        printf("Ingresa una opcion valida");
    }

    }

}

File: Proyectos_python/menu_while_dowhile.py
while True:
    print ("Puedo calcular areas de diferentes figuras, cuadrado, triangulo y circulo.")
    print ("selecciona una opcion")
    print ("1.Area de cuadrado\n2.Area de triangulo\n3.Area de circulo\n4.Salir")
    opcion=int(input())
    if opcion == 1: 
        print("Ingresa la medida de un lado de tu cuadrado")
        lado=int(input())
        Area_cuadrado=lado*lado
        print(f"El area de tu cuadrado es:{Area_cuadrado}")
    elif opcion == 2:
        print("Infresa la base de tu triangulo")
        base=int(input())
        print("Ingresa la altura de tu triangulo")
        altura=int(input())
        Area_triangulo=(base*altura)/2
        print(f"El area de tu triangulo es:{Area_triangulo}")
    elif opcion == 3:
        print("Ingresa el radio de tu circulo")
        pi=3.14159265359
        radio=int(input())
        Area_circulo=pi*(radio*radio)
        print(f"El area de tu circulo es{Area_circulo} ")
    elif opcion == 4:
        print("Adios!")
        break
    else:
        print ("Opcion no valida, por favor elige una opcion valida")
    